src/components/Button/Button.stories.tsx

```diff
@@ -1,26 +1,29 @@
 import { ComponentMeta, ComponentStory } from '@storybook/react';
 import { Button, ButtonVariant } from './Button';
+import { WithGrayBackground } from '../../hoc/WithGrayBackground';
 
 export default {
   title: 'Components/Button',
   component: Button,
 } as ComponentMeta<typeof Button>;
 
-const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
+const ButtonTemplate: ComponentStory<typeof Button> = (args) => (
+  <WithGrayBackground>
+    <Button {...args} />
+    <Button {...args} />
+  </WithGrayBackground>
+);
 
-export const Primary = Template.bind({});
+export const Primary = ButtonTemplate.bind({});
 Primary.args = {
   children: 'Children',
+  variant: ButtonVariant.PRIMARY,
+  onClick: () => console.log('Clicked'),
 };
 
-export const Rounded = Template.bind({});
-Rounded.args = {
+export const Secondary = ButtonTemplate.bind({});
+Secondary.args = {
   ...Primary.args,
-  rounded: true,
-};
-
-export const Danger = Template.bind({});
-Danger.args = {
-  ...Primary.args,
-  variant: ButtonVariant.DANGER,
+  variant: ButtonVariant.SECONDARY,
+  onClick: () => console.log('Clicked'),
```

Comment:
`onClick: () => console.log('Clicked'),` is already part of `...Primary.args,`, so should be removed